addons/stock/i18n/stock.pot

```diff
@@ -1099,18 +1094,6 @@ msgid "Dates of Inventories and latest Moves"
 msgstr ""
 
 #. module: stock
-#: model:res.company,overdue_msg:stock.res_company_1
-msgid "Dear Sir/Madam,\n"
```

Comment:
why removing this one? It is [still present](https://github.com/odoo/odoo/blob/d1d4688521b919441b4d2e6924df2d21b5046dee/addons/account/company.py#L40)